src/daos/job.js

```diff
@@ -62,7 +62,11 @@ class JobDao {
    */
   async fetchJobById(jobId) {
     const jobRow = await this.JobStore.fetchJobById(jobId);
-    return this._JobRowtoJobData(jobRow);
+    if (jobRow !== undefined) {
+      return this._JobRowtoJobData(jobRow);
+    } else {
+      return null;
+    }
```

Comment:
Please change this to use the return early approach instead.

Reply:
```
async fetchJobById(jobId) {
    const jobRow = await this.JobStore.fetchJobById(jobId);
    if (jobRow === undefined) {
      return null;
    }
    return this._JobRowtoJobData(jobRow);
  }
```